.github/workflows/documentation.yml

```diff
@@ -0,0 +1,66 @@
+name: Build documentation
+
+on:
+  schedule:
+    - cron: '0 0 * * SAT' # every Saturday
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout blog
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.USER_TOKEN }}
+
+      - name: Install OCaml
+        uses: avsm/setup-ocaml@v1
+        with:
+          ocaml-version: 4.09.1
+
+      - name: Pin OASIS
+        run: opam pin add oasis https://github.com/BinaryAnalysisPlatform/oasis.git
+
+      - name: Checkout bap
+        uses: actions/checkout@v2
+        with:
+          repository: BinaryAnalysisPlatform/bap
+          path: bap.master
+
+      - name: Pin BAP
+        run: opam pin add bap bap.master --no-action
+
+      - name: Install system dependencies
+        run: opam depext -u bap
+
+      - name: Install opam dependencies
+        run: |
+            opam install bap --deps-only
+            opam install omake
+
+      - name: Install Bap
+        run: |
+          cd bap.master
+          opam exec -- ./configure-omake
+          opam exec -- make
+          opam exec -- make reinstall
+          cd ../
+
+      - name: Build Documentation
+        run: |
+          sudo apt-get install emacs-nox man2html --yes
+          opam install odig --yes
+          opam exec -- make -C bap.master clean
+          opam exec -- make -C bap.master doc
+
+      - name: update and push
+        run: |
+          git config --global user.name ${{ secrets.USER_NAME }}
+          git config --global user.email ${{ secrets.USER_EMAIL }}
+          cp -r  bap.master/doc/man1 bap/api/
```

Comment:
copying will take a lot of time and we don't really need copies. Just use `mv`, e.g., `mv bap/master/doc/man1 bap/api/man1`